packages/@ama-sdk/schematics/schematics/migrate/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Rule, type Tree } from '@angular-devkit/schematics';
+import { MigrateSchematicsSchemaOptions } from './schema';
+import { getMigrationRuleRunner, type MigrationRulesMap } from '@o3r/schematics';
+import { resolve } from 'node:path';
+
+const tsMigrationMap: MigrationRulesMap = {
+
+};
+
+const isTypescriptSdk = (tree: Tree) => {
+  return tree.exists('/tsconfig.json');
```

Comment:
This is not enough for applications with tsconfig files following the tsconfig.bas.json pattern (you can have a look at https://github.com/AmadeusITGroup/otter/pull/1696)

Reply:
You should always have a `tsconfig.json` in anycase, if there is no `tsconfig.json` in a repository based on Typescript this is mainly an issue (maybe #1696 should be fixed then :D).

But indeed, we can be more safe by checking the file pattern `tsconfig*.json`